app/controllers/application_controller.rb

```diff
@@ -3,6 +3,15 @@ class ApplicationController < ActionController::Base
 
   include SessionsHelper
 
+  def log_in(user_id)
+    session[:user_id] = user_id
+  end
+
+  def logout
+    session.clear
+    @current_user = nil
+  end
+
```

Comment:
**Code Quality - Nitpick**
Move `log_in` and `log_out` to application controller since they are not used in the views.